### ISSUE TITLE
XEP-0363

### ISSUE THREAD

**Issue by stephendaniel** [zom/Zom-Android-XMPP#198]
Hi, is it possible to use XEP-0363 HTTP Upload in Zom Android?

**Comment by mimi89999** [comment]
@stephendaniel No. AFAIK, XEP-0363 is not supported. Also, file sharing is tracked in #16.

**Comment by n8fr8** [comment]
We are planning to implement this in May, to be compatible with Conversations, ChatSecure, etc.
